### PR TITLE
[3221][ADD] account_move_ref_duplicatable

### PR DESCRIPTION
**Pull request by rinaldifirdaus**
[3221](https://www.quartile.co/web?db=quartile&token=dMRr78CMkEX6gyy9nTiv#id=3221&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: account_move_duplicate_ref/models/account_move.py
Comment (yostashiro) [inline account_move_duplicate_ref/models/account_move.py:1]: @rinaldifirdaus I was thinking that we would need this, but is it not the case?
```
    ref = fields.Char(copy=True)
```

File: account_move_duplicate_ref/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    _inherit = ["account.move"]
+
+    @api.returns("self", lambda value: value.id)
+    def copy(self, default=None):
+        self.ensure_one()
+        if default is None:
+            default = {}
+        if "ref" not in default:
+            default["ref"] = _("%s (copy)", self.ref)
+        return super(AccountMove, self).copy(default=default)
```
Comment (yostashiro) [inline account_move_duplicate_ref/models/account_move.py]:
```suggestion
        return super().copy(default=default)
```

File: account_move_duplicate_ref/models/account_move.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from odoo import _, api, models
+
+
+class AccountMove(models.Model):
+    _inherit = ["account.move"]
```
Comment (yostashiro) [inline account_move_duplicate_ref/models/account_move.py]:
```suggestion
    _inherit = "account.move"
```

**Review by yostashiro — CHANGES_REQUESTED** [review]
Can we rename the module to `account_move_ref_duplicatable`?

File: account_move_duplicate_ref/models/account_move.py
Comment (rinaldifirdaus) [reply account_move_duplicate_ref/models/account_move.py:1]: With current code, i've tested that even we don't add "ref = fields.Char(copy=True)" the value was properly copied, Tashiro san.

**Comment by rinaldifirdaus** [comment]
@yostashiro could you please help to merge this PR? client has confirm that we can install this to production. Thank you

**Review by yostashiro — APPROVED** [review]

**Pull request merged**